server/controllers/availability.js

```diff
@@ -0,0 +1,98 @@
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+const Schedule = require("../models/Schedule");
+
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+  const profileId = req.profile.id;
+  const name = req.body.name;
+  const days = req.body.days;
```

Comment:
Nitpick: You can you a destructuring assignment here. You can also validate each and send a `400` if any is missing.